app/default_overrides_test.go

```diff
@@ -86,9 +85,9 @@ func TestDefaultConsensusConfig(t *testing.T) {
 			// Overrides
 			MaxTxBytes:   7_897_088,
 			MaxTxsBytes:  39_485_440,
-			TTLDuration:  75 * time.Second,
-			TTLNumBlocks: 12,
-			Version:      "v1",
+			TTLDuration:  150 * time.Second,
+			TTLNumBlocks: 24,
```

Comment:
I expect this test to fail b/c these values diverge from what is specified in `default_overrides.go`